src/commandDetails/coin/leaderboard.ts

```diff
@@ -38,7 +38,16 @@ const getCurrentCoinLeaderboardEmbed = async (
     }
     if (user.bot) continue;
     const userTag = user?.tag ?? '<unknown>';
-    const userCoinEntryText = `${leaderboardArray.length + 1}. ${userTag} - ${
+    const cleanUserTag = userTag
+      .split('~')
+      .join('\\~')
+      .split('*')
+      .join('*')
```

Comment:
This should be `\\*` instead?

Reply:
darn i wanted to call yall out for skim review 💀